app/Seeder.cs

```diff
@@ -91,7 +91,7 @@ private static IEnumerable<TodoItem> CreateTodoItems(IList<Tag> tags, IList<Pers
             var assigneeIterator = new LoopingCollectionIterator<Person>(people, 2);
             var tagIterator = new LoopingCollectionIterator<Tag>(tags);
 
-            var creationTime = new DateTimeOffset(2000, 1, 1, 1, 1, 1, 1, TimeSpan.FromHours(5));
+            var creationTime = new DateTimeOffset(2000, 1, 1, 1, 1, 1, 1, TimeSpan.Zero);
```

Comment:
Npgsql doesn't support writing DateTimeOffset with non-zero offset. More info can be found there https://www.roji.org/postgresql-dotnet-timestamp-mapping.